src/loops/programs_with_repetition/easy/q4/ItsAlwaysPrimeTime.java

```diff
@@ -0,0 +1,30 @@
+package loops.programs_with_repetition.easy.q4;
+
+/*
+ * Write a program to keep asking the user to enter a number until they enter a prime number that is lesser than 50. Make sure your output matches the sample. 
+ * 
+ * Sample output:
+ *    Enter an integer:
+ *    2
```

Comment:
Again, we should probably have some invalid input in the sample so the student can see that "Enter an integer:"
 is displayed multiple times.